packages/sui-ssr/build-device.js

```diff
@@ -1,8 +1,13 @@
-export const buildDeviceFrom = ({request = {}, window = {}}) => {
+export var buildDeviceFrom = function(params) {
+  var win = params.window
+  var request = params.request
+  request = request === undefined ? {} : request
```

Comment:
Am I right that we can change it by this?

var request = params.request || {}

Same with window? Or the undefined explicit comparision have a purpose?

Reply:
The explicit comparision is to keep the exact same behavior as before for default value in param function. The one you're mentioned is a loose, as is checking a falsy value instead. In my opinion, both could work for this case.